internal/checks/promql_offset_test.go

```diff
@@ -0,0 +1,92 @@
+package checks_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cloudflare/pint/internal/checks"
+	"github.com/cloudflare/pint/internal/promapi"
+)
+
+func newOffsetCheck(_ *promapi.FailoverGroup) checks.RuleChecker {
+	dur, _ := time.ParseDuration("5m")
+	return checks.NewOffsetCheck(checks.MustTemplatedRegexp("aws_.*"), dur)
+}
+
+func TestOffsetCheck(t *testing.T) {
```

Comment:
Should we add recording rule test(s) just to be sure that's also working?

Reply:
Good idea